test/helper.rb

```diff
@@ -90,6 +90,9 @@ def assert_array_in_delta(expected_array, actual_array, delta = 0.001, message =
 require 'vcr'
 
 VCR.configure do |c|
+  c.before_record do |i|
+    i.response.body.force_encoding('UTF-8')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.